lte/gateway/c/session_manager/test/test_upf_msg_manage_handler.cpp

```diff
@@ -128,7 +245,20 @@ TEST_F(SetUPFNodeState, test_upf_session_config) {
      * with SMF latest version
      */
     EXPECT_LE(version, cur_version);
+
+    EXPECT_TRUE(
+        session_enforcer->is_incremented_rtx_counter_within_max(session));
+
     RulesToProcess pending_activation, pending_deactivation;
+    const CreateSessionResponse& csr = session->get_create_session_response();
+    std::vector<StaticRuleInstall> static_rule_installs =
```

Comment:
Can you also rename the file here as well? https://github.com/magma/magma/blob/master/lte/gateway/c/session_manager/test/BUILD.bazel#L261

You can verify the Bazel testing by running this in the Magm VM.

```bash
cd magma/
bazel test //lte/gateway/c/session_manager/test:all
```

Reply:
renamed the file `test_upf_node_state.cpp` to `test_upf_msg_manage_handler.cpp` under `magma/lte/gateway/c/session_manager/test/BUILD.bazel` . Also added one more UT file `test_session_state_5g.cpp` which got missed under `magma/lte/gateway/c/session_manager/test/BUILD.bazel`. Below is the screen shot for passing test cases for Bazel.
![image](https://user-images.githubusercontent.com/70899516/140557230-cc1dba60-002d-4cc3-922a-c589b6bb72d1.png)